src/clj/game/cards/operations.clj

```diff
@@ -235,22 +235,26 @@
              card nil)))}})
 
 (defcard "Audacity"
-  (letfn [(audacity [n]
-            (when (< n 2)
-              {:prompt "Choose a card to place advancements tokens on"
-               :async true
-               :choices {:card can-be-advanced?
-                         :all true}
-               :msg (msg "place 1 advancement token on " (card-str state target))
-               :effect (req (add-prop state :corp target :advance-counter 1 {:placed true})
-                            (continue-ability state side (audacity (inc n)) card nil))}))]
+  (letfn [(audacity
+             [x] {:prompt (msg "Choose a card that can be advanced to place advancement counters on (" x " remaining)")
+                  :async true
+                  :choices {:card can-be-advanced?
+                            :all true}
+                  :msg (msg "place 1 advancement counter on " (card-str state target))
+                  :effect (req (wait-for (add-prop state side target :advance-counter 1 {:placed true})
+                                    (if (> x 1)
```

Comment:
```suggestion
                                    (if (pos? x)
```

Reply:
On a second thought, this is not correct: Audacity inner function would be called 3 times while it should only be called twice.